src/database.cpp

```diff
@@ -310,14 +310,16 @@ bool Database::load()
          }
    }
 
-   // With the db open, we can get all our hashes. This has to happen before
-   // updateSchema
-   Database::tableNames = tableNamesHash();
-   Database::classNameToTable = classNameToTableHash();
```

Comment:
It is now. I will fix.

Mik
On Mar 5, 2016 12:45, "Philip Lee" <notifications@github.com> wrote:

> In src/database.cpp
> <https://github.com/Brewtarget/brewtarget/pull/207#discussion_r55124728>:
>
> > @@ -310,14 +310,16 @@ bool Database::load()
> >           }
> >     }
> >
> > -   // With the db open, we can get all our hashes. This has to happen before
> > -   // updateSchema
> > -   Database::tableNames = tableNamesHash();
> > -   Database::classNameToTable = classNameToTableHash();
>
> Is the comment
>
> This has to happen before updateSchema
>
> just wrong?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Brewtarget/brewtarget/pull/207/files#r55124728>.
>